WoWsShipBuilder.Web/Dialogs/CustomAccelerationDataDialog.razor

```diff
@@ -164,4 +164,17 @@
         MudDialog.Close(DialogResult.Ok(container));
     }
 
+    private bool EnableOkButton()
```

Comment:
Rename this, in this cased when it is used for the Ok Button's Disabled parameter, use a name like IsOkButtonDisabled and adjust logic accordingly